devices/src/acpi.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright © 2019 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use vmm_sys_util::eventfd::EventFd;
+use BusDevice;
+
```

Comment:
Here as well: Fine as a first iteration, but the whole ACPI support, including this ACPI device, should be made a build time option.